### PR TITLE
Guard against ignite new ignite

### PR DESCRIPTION
**Pull request by fvonhoven**
## Describe your PR
Addresses #975 
Guard against igniting a new project with name `ignite` or `Ignite`. 👍

### CODE REVIEW

File: src/commands/new.js [hunk 1]
```diff
@@ -27,6 +27,12 @@ async function command (context) {
   // camelCase the project name for user example
   const projectNameCamel = upperFirst(camelCase(projectName))
 
+  // Guard against `ignite new ignite`
+  if (projectNameCamel === 'ignite') {
```
Comment (jamonholmgren) [inline src/commands/new.js]: I'm pretty sure this won't work (it'll be `Ignite` not `ignite`) ... can you test @fvonhoven ?

Comment (fvonhoven) [inline src/commands/new.js]: @jamonholmgren 
alt tag

**Comment by GantMan** [comment]
failing semaphore?

**Comment by fvonhoven** [comment]
@GantMan alt tab

File: src/commands/new.js [hunk 2]
```diff
@@ -47,6 +47,12 @@ async function command (context) {
     process.exit(exitCodes.PROJECT_NAME)
   }
 
+  // Guard against `ignite new ignite`
+  if (toLower(projectName) === 'ignite') {
+    context.print.error('Crossing the streams is bad! Please use another name for your project.')
```
Comment (jamonholmgren) [inline src/commands/new.js]: Not super keen on this wording, @fvonhoven, it might be a little confusing. Any other ideas?

**Pull request merged**